Source/Core/VideoCommon/VertexLoaderX64.cpp

```diff
@@ -57,11 +57,10 @@ VertexLoaderX64::VertexLoaderX64(const TVtxDesc& vtx_desc, const VAT& vtx_att)
 
 OpArg VertexLoaderX64::GetVertexAddr(CPArray array, VertexComponentFormat attribute)
 {
-  OpArg data = MDisp(src_reg, m_src_ofs);
```

Comment:
It's two copies of the `MDisp` call for this (and `GameFile.cpp`/`VolumeVerifier.cpp`, but they likely aren't as relevant to performance as the vertex loader is) vs. one copy in the old code - is this _really_ worth enough to pick copy&paste instead? Is (N)RVO this crazy good/important here?

My threshold is normally having 3 copies before I consider restructuring things, but this just feels....weird.
I don't want to hold this up if I'm the only one though, since the only thing I can think of is simply _not_ changing it at this point 🤷

Reply:
It's definitely a micro-optimization (a matter of copying 24 bytes out from a function called maybe tens of times during scene transitions). To argue in favor of this change: I felt the content of what is being copy-pasted here was so short that it's not unlike giving it a more descriptive identifier than "data".

Reply:
Fair, I _did_ remove a sentence about "being more readable" before sending my review. Still feels odd, but I suppose you have a point.